pkg/ui/constants.go

```diff
@@ -15,7 +15,7 @@ const (
 	AlertMetadataViewTitle    = "[ ALERT DATA ]"
 	IncidentsTableTitle       = "[ TRIGERRED INCIDENTS ]"
 	AckIncidentsTableTitle    = "[ ACKNOWLEDGED INCIDENTS ]"
-	OncallTableTitle          = "[ ONCALL ]"
+	OncallTableTitle          = "ONCALL"
```

Comment:
Any reason why it's not called `[ ONCALL ]`?

Reply:
The overall table title appears a `[ ONCALL Layer 5 [ NASA-W ] ]` 
This is set when we initialize the tables. It's not called [ONCALL] because then the title will not appear properly on the table. It will be somewhat like `[ ONCALL ][ Layer 5 [ NASA-W ] ]`